src/en/howto-windows.md

```diff
@@ -0,0 +1,85 @@
+## This guide is currently experimental and under dev
+
+Juju is capable of orchestrating Windows workloads thanks to the enablement
+efforts of [CloudBase Solutions](http://cloudbaseit.com).
+
+> Getting started with orchestrating windows workloads today is still a work in
+progress - as it's only available on 2 substrates. MAAS 1.8 experimental, and
+OpenStack with a custom image loaded in Cinder.
+
+<!-- This may be completely overkill - but i wanted to be somewhat complete
+if we were going with a MAAS + Juju approach, as thats the most tested
+integration that we have today. If this doesn't belong here - feel free to
+punt -->
+
+### Set up MAAS
+
+You will need to add the MAAS experimental PPA
+
+    sudo add-apt-repository ppa:~maas-maintainers/experimental
```

Comment:
We have custom image support in stable? I'll have to take a look - I haven't dove too deep into this however. I'll take a gander and update as required